LiteFarmOrg/LiteFarm
# LF-2163 and LF-2172  Addition of "Needs plan" counter in Crop catalogue and crop variety

Important! - this PR contains UI changes related to Lf-2163 and LF-2172. Please review the LF-2172 PR (https://github.com/LiteFarmOrg/LiteFarm/pull/1900)
and then this one.

To test:

- create a crop variety by selecting any crop from the crop catalog page. (http://localhost:3000/crop_catalogue)
- add more varieties for the same crop. (around 5)
- add a plan to any of the crop varieties.
- check for the count of active, planned, past, and needs plans.

For more details:
https://lite-farm.atlassian.net/browse/LF-2163
https://lite-farm.atlassian.net/browse/LF-2172
